spec.bs

```diff
@@ -3708,19 +3719,35 @@ Each {{InterestGroupBiddingScriptRunnerGlobalScope}} has a
 :: A [=boolean=]
 : <dfn>group has ad components</dfn>
 :: A [=boolean=]
+: <dfn>multi-bid limit</dfn>
+:: An integer.

```

Comment:
Is it better to use {{unsigned short}} for this as well? Although it's defined in webidl, we also used types like {{double}} in the spec.

Reply:
Done